src/components/Modal/src/ModalLayer.vue

```diff
@@ -114,6 +128,36 @@ export default {
 	right: 0;
 	bottom: 0;
 	left: 0;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+}
+
+.Translucent {
+	position: fixed;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	background-color: rgba(0, 0, 0, 0.3);
+}
+
+.ModalContainer {
+	width: 100%;
+	height: 100%;
+	overflow: scroll;
+	box-shadow: 0 0 24px 8px rgba(0, 0, 0, 0.3);
+}
+
+@media screen and (min-width: 1200px) {
+	.ModalContainer {
+		display: inline-block;
+		width: auto;
+		max-width: 1000px;
+		height: auto;
```

Comment:
Not sure yet how we're going to handle modal stacking and the height potentially changing, but I'm guessing CSS won't be enough and we may need to pass events about the height changing to keep "parent" modals in sync.

Reply:
Why do we need to keep them in sync? Also would this not be the responsibility of the end user? They know the size requirements of their modals better than we do.

Reply:
I've been assuming based on our conversations with design that we'd want to constrain the proportions of the parent and child so that the child is contained inside the parent.  However, if a child needs to be bigger / smaller than the parent we'd communicate to the parent its size so we could transition smoothly when the child is closed to reset the parent back to it's original size.

Maybe that's too much to assume. Maybe that's too much to manage. But if that's the desired behavior we'd have to either do it internally or provide hooks / documentation for the dev to do it themselves.

Reply:
solved with TransitionResize component